swift-ios/Task Tracker/Models.swift

```diff
@@ -26,6 +26,17 @@ class Project: Object {
     }
 }
 
+class User: Object {
+    @objc dynamic var _id: ObjectId = ObjectId.generate()
+    @objc dynamic var _partition: String? = nil
+    @objc dynamic var image: String? = nil
+    @objc dynamic var name: String = ""
+    @objc dynamic var user_id: String = ""
+    override static func primaryKey() -> String? {
+        return "_id"
+    }
+}
+
```

Comment:
I'm guessing this is a piece of custom user data? Or is the user meant to fetch this via Realm sync?

Reply:
This will be used, and is used in the GraphQL tutorial, but as long as it's defined in the backend, it must be specified here.